### PR TITLE
Krsd9 2nd

### PR DESCRIPTION
**Pull request by KRS-Newberg**
Hopefully this will work out better. I forgot 90% of my git in the past few weeks

### CODE REVIEW

File: script/Arcueid/Day 9/09_00_ARC09_1.txt [hunk 1]
```diff
@@ -147,7 +148,7 @@ I'm gazing down at the scene before me.
 [sha:5a3b9ab142cbd637aef5fa4d5995d6b55c2a071c]{
 -- Page 5, Offset 21321.
 -- 両手は倒れている子供と同じように赤い。
-My hands are red. Just like the child, collapsed in a heap on the floor.
+My hands are red, just like the child, collapsed in a heap on the floor.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:151]: Alright, so originally I'd had this line pair as 
> My hands are red, just like the child now collapsed in a heap on the floor.
> My hands are red, with the blood of the living being now collapsed in a heap on the floor.

But it was suggested that maybe that's a bit wordy, and the commas were proposed. _However_, I feel that the comma in the second sentence of the pair is kinda easy to parse wrong, as though it is the hands that are collapsed:
> My hands are red with the blood of a living being, collapsed in a heap on the floor.

I don't really love the full stop, but it does at least separate out the sentence better. I might push for either 
1. going back to `now collapsed` with only one comma (after red)
2. breaking it with a full stop, but restating red to make it less abrupt, i.e.

> My hands are red. Red like the child collapsed in a heap on the floor.
> My hands are red. Red with the blood of the living being collapsed in a heap on the floor.

File: script/Arcueid/Day 9/09_00_ARC09_1.txt [hunk 2]
```diff
@@ -26,6 +26,7 @@ Things that my father ordered me to forget.
 [sha:2ce4023198d762d748648342d653f8bf32994ce0]{
 -- Page 2, Offset 21297.
 -- 夏の、暑い日。
+//A hot summer day is used more commonly in prose
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:29]: ngrams seems to agree. The linter is mad because this line is used elsewhere, so just be sure to update 11_01_CIEL11_1 to match.

File: script/Arcueid/Day 9/09_00_ARC09_1_1.txt [hunk 3]
```diff
@@ -8,6 +8,7 @@ I mean, how could she not stand out?
 [sha:6b2e3c7edac8deb554a5096542c3913a8ab5473b]{
 -- Page 10001, Offset 21351.
 -- 昼間の学校強襲から始まって、夜の巡回まで、あいかわらずさんざん振り回された。
+//'wrapped around her finger' implies manipulation just wan't to check the JP
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1_1.txt:11]: Yup
https://ejje.weblio.jp/content/%E6%8C%AF%E3%82%8A%E5%9B%9E%E3%81%95%E3%82%8C%E3%82%8B

File: script/Arcueid/Day 9/09_00_ARC09_2.txt [hunk 4]
```diff
@@ -70,6 +70,7 @@ I should save that for tonight and discuss them with Arcueid.
 [sha:872894ed81cd6df96c42e8e937e7ac2c51c35f39]{
 -- Page 30, Offset 21394.
 -- 昼間ぐらいは自分の日常に戻らないと、俺もどうにかしてしまう。
+//Instinctually, I would never use drastic here. In the english absurd sounds good?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2.txt:73]: As in, replace drastic with absurd? I think the intent here is "I'll do something I might regret later", so absurd doesn't really fit in that context. Possible "do something drastic" is UK speak, if so you could maybe naturalize it to "it might really push me over the edge" or something.

File: script/Arcueid/Day 9/09_00_ARC09_2_1.txt [hunk 5]
```diff
@@ -192,6 +192,7 @@ Shiki-chan, could you whip those clothes off for me?"
 [sha:1d934bd0ca78faffeafa2e2707eb0961b6af3de3]{
 -- Page 61, Offset 21467.
 -- 見たいのは志貴チャンの上半しーん！　うふふ……ソチラ、胸にすっごい傷跡があるんじゃなーい？」
+//seems like there is about a 50/50 spread 'upper-body' vs 'upper body'. Your choice
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2_1.txt:195]: I'd assume upper-body is more when used as an adjective?

File: script/Arcueid/Day 9/09_00_ARC09_2_2.txt [hunk 6]
```diff
@@ -144,11 +144,12 @@ And besides, look.
 [sha:f102a5486a307353a5970cd75747cb4eeb8d808d]{
 -- Page 82, Offset 21517.
 -- 「うん、鞄に入れてきたものだけで十分。
-"Nope, the things I put in my bag are all I need.
+"Nope, the things I put in my bag werw all I needed.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2_2.txt:147]: Typo

File: script/Arcueid/Day 9/09_00_ARC09_2_2.txt [hunk 7]
```diff
@@ -144,11 +144,12 @@ And besides, look.
...
 }
 [sha:8c629cec38a5f4465c5a53b2c5548018a479a8aa]{
 -- Page 82, Offset 21518.
 -- 大事なものは、ここに来る時に全部持ってきた」
+//There is an arguement to be made for held dear but I think it's fine as is
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_2_2.txt:152]: Cool. When the time comes to merge, be sure to go back and clean up any comments that are specific to this review, just so they don't confuse use later.

File: script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt [hunk 8]
```diff
@@ -139,7 +139,8 @@ But, unbelievable as it is, the thing in front of me now is definitely real.
 [sha:44fe03efbe94d42619e47b605eb1a0214ccd2202]{
 -- Page 161, Offset 21748.
 -- 槙久の部屋の窓は、一面の壁だった。
-The window in Makihisa's room has been completely walled off.
+//Walled off refers more closely to literal walls 
+The window in Makihisa's room has been completely sealed shut.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt:143]: I mean the CG is pretty heavy duty, I might lean towards keeping it walled.

File: script/Arcueid/Day 9/09_00_ARC09_4_2.txt [hunk 9]
```diff
@@ -971,6 +971,7 @@ I give her a wave and set off home.
 [sha:6b5eec75a1a0b63e27e08afe1e236fcc0d330c93]{
 -- Page 299, Offset 22124.
 -- と、アルクェイドに呼び止められた。
+//instead of 'calls me to a halt' consider 'brings me to a halt' or 'stops me in my tracks'
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_4_2.txt:974]: Hmm, she is doing it at a distance by calling out to him, so I think it might be ok? The other two feel a bit more physically arresting, at least to me.

File: script/Arcueid/Day 9/09_00_ARC09_1.txt (hunk 1 above)
Comment (Avuxo) [reply script/Arcueid/Day 9/09_00_ARC09_1.txt:151]: Of those two proposals, I favor "My hands are red. Red like the child collapsed in a heap on the floor.". With that said, I don't think anybody is gonna read it as the _hands_ being collapsed. There's enough context that it's not an easy mistake.

File: script/Arcueid/Day 9/09_00_ARC09_2.txt (hunk 4 above)
Comment (Avuxo) [reply script/Arcueid/Day 9/09_00_ARC09_2.txt:73]: I think the current TL is good for どうにかしてしまう。

File: script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt (hunk 8 above)
Comment (Avuxo) [reply script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt:143]: idk, I think I agree with this edit. I can't really explain it other than "walled off" makes me think of the level of Ghost Master where the old lady has sealed a welder behind literal bricks.

The CG _is_ more sealed off though for a more coherent explanation

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 9/09_00_ARC09_1.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 9/09_00_ARC09_1.txt:151]: Yeah, I don't think anyone will leave the sentence with the wrong impression, it's just a bit of a hiccup.

File: script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt (hunk 8 above)
Comment (rschlaikjer) [reply script/Arcueid/Day 9/09_00_ARC09_3_2_1.txt:143]: For the love of god, Montresor.

File: script/Arcueid/Day 9/09_00_ARC09_1.txt [hunk 10]
```diff
@@ -157,7 +157,7 @@ No, that's not right.
 [sha:05745a7508e4c6d8f5598d3c09aed72c31ff9f81]{
 -- Page 5, Offset 21323.
 -- この両手は、倒れている<子供|イキモノ>の血で赤いのだ。
-My hands are red. With the blood of a living being, collapsed in a heap on the floor.
+My hands are red with the blood of a living being, collapsed in a heap on the floor.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:160]: Gotta update this one too

File: script/Arcueid/Day 9/09_00_ARC09_4_2.txt (hunk 9 above)
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_4_2.txt:974]: Delete comment

File: script/Arcueid/Day 9/09_00_ARC09_1.txt [hunk 11]
```diff
@@ -157,7 +157,7 @@ No, that's not right.
 [sha:05745a7508e4c6d8f5598d3c09aed72c31ff9f81]{
 -- Page 5, Offset 21323.
 -- この両手は、倒れている<子供|イキモノ>の血で赤いのだ。
-My hands are red. With the blood of a living being, collapsed in a heap on the floor.
+My hands are red. Red with the blood of a living being, collapsed in a heap on the floor.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 9/09_00_ARC09_1.txt:160]: Sorry, there were two more changes to make it match nicely - s/a/the and drop the comma
> My hands are red. Red with the blood of **the** living being collapsed in a heap on the floor.

**Review by rschlaikjer — APPROVED** [review]
